target-chains/cosmwasm/tools/src/pyth_config.ts

```diff
@@ -0,0 +1,132 @@
+import { NETWORKS } from "./network";
+
+export type PythConfig = {
+  wormhole_contract: string;
+  data_sources: DataSource[];
+  governance_source: DataSource;
+  governance_source_index: number;
+  governance_sequence_number: number;
+  chain_id: number;
+  valid_time_period_secs: number;
+  fee: Fee;
+};
+
+export type DataSource = {
+  emitter: string;
+  chain_id: number;
+};
+
+export type Fee = {
+  amount: string;
+  denom: string;
+};
+
+type Config = Record<NETWORKS, PythConfig>;
+
+export const PythConfig: Config = {
+  [NETWORKS.TERRA_MAINNET]: {
+    wormhole_contract:
+      "terra12mrnzvhx3rpej6843uge2yyfppfyd3u9c3uq223q8sl48huz9juqffcnh",
+    data_sources: [
+      {
+        emitter: Buffer.from(
+          "6bb14509a612f01fbbc4cffeebd4bbfb492a86df717ebe92eb6df432a3f00a25",
+          "hex"
+        ).toString("base64"),
+        chain_id: 1,
+      },
+      {
+        emitter: Buffer.from(
+          "f8cd23c2ab91237730770bbea08d61005cdda0984348f3f6eecb559638c0bba0",
+          "hex"
+        ).toString("base64"),
+        chain_id: 26,
+      },
+    ],
+    governance_source: {
+      emitter: Buffer.from(
+        "5635979a221c34931e32620b9293a463065555ea71fe97cd6237ade875b12e9e",
+        "hex"
+      ).toString("base64"),
+      chain_id: 1,
+    },
+    governance_source_index: 0,
+    governance_sequence_number: 0,
```

Comment:
@ali-bahjati question for you: what are we supposed to set as the default values for these two fields? I set them as zero before but we've probably generated a couple governance messages already, right?

Reply:
Yeah but I think using 0 as the beginning is fine. There is no harm executing older ones.